R/GroupCoverages.R

```diff
@@ -28,6 +32,7 @@ addGroupCoverages <- function(
   ArchRProj = NULL,
   groupBy = "Clusters",
   useLabels = TRUE,
+  sampleLabels = NULL,
```

Comment:
Set default to "Sample" just because you are doing that anyways and dont allow it to be null is a safer setup i think